tensorflow/python/framework/function.py

```diff
@@ -748,6 +748,9 @@ def foo(x, y):
   default graph and adds the definition of the function into the
   default graph. Because the addition of the function into the graph
   is deferred, the decorator can be used anywhere in the program.
+  
+  Definitions of function is frozen for the session after first session.run
```

Comment:
That doesn't seem to match current behavior -- I can create a session, then create function node, and then issue first session.run and it works, still want me to change it to that?